docs/source/backends/llamacpp.md

```diff
@@ -54,6 +54,10 @@ cd ~/models
 curl -LOJ "https://huggingface.co/Qwen/Qwen2.5-3B-Instruct-GGUF/resolve/main/qwen2.5-3b-instruct-q4_0.gguf?download=true"
 ```
 
```

Comment:
You probably can remove the model preparation step as you now auto-prepare the model and add a `## Using a custom GGUF` section after the `## Run Docker image`? Idea is to have the least steps in the doc for a first "quick try". wdyt?